src/BackgroundImage.js

```diff
@@ -1,26 +1,29 @@
 import styled from 'styled-components'
 import PropTypes from 'prop-types'
+import { width, responsiveStyle } from 'styled-system'
```

Comment:
Why do you import `responsiveStyle` if you don't use it?

Reply:
removed